HACKING.adoc

```diff
@@ -575,14 +575,39 @@ runs much faster than a full `./tools/check-typo`, typically instantly.
 
 You can also setup a git commit-hook to automatically run `check-typo`
 on the changes you commit, by copying the file
-`tools/pre-commit-githook` to `.git/hooks/pre-commit`. If changes in a commit
-alter the `configure` script, the hook also checks that committed `configure`
-script is up-to-date.
+`tools/pre-commit-githook` to `.git/hooks/pre-commit`. If changes in a
+commit alter the `configure` script, the hook also checks that
+committed `configure` script is up-to-date. The `configure` script may
+be regenerated by running `make -B configure`, with GNU Autoconf
```

Comment:
Isn't `tools/autogen` preferred to `make configure`? (See e.g. `VERSION`)

Reply:
The CI uses `make -B configure` and the Makefile has:

```makefile
configure: tools/autogen configure.ac aclocal.m4 build-aux/ocaml_version.m4
	$<
```

Reply:
and the pre-commit hook says:
> `configure.ac` does not appear to generate configure
> Try running `make -B configure` and stage the changes

Reply:
Then it would be nice to update `VERSION` to mention `make -B configure` as well, for consistency.